Add Beneath Cloud Top Manor

Sabre142 opened the pull request.

Sabre142 committed:
Add Beneath Cloud Top Manor

* Add mod entry
 - To the Locations - Player homes section.

* Add location
 - https://www.nexusmods.com/oblivion/mods/46148/
 - Add mod name.

* Add requirement
 - Missing SI master.

* Add message
 - Wild edit in 00037331. Unnecessarily renames a record that is unused
   in this mod.

* Add tag suggestions
 - C.Water, modifies water height in 00007983.
 - Graphics, modifies model in 0003753C.
 - Relations.Add, adds faction relations in 0009DB1F.
 - Relations.Remove, removes faction relations in 0009DB1F.
 - Scripts, modifies the script in 0003753C.

* Add cleaning data
 - Version 2.0:
 - Pre-WE: 4E83861A.

Sabre142 committed:
Add Beneath Cloud Top Manor

* Add requirement
 - Requires OBSE.

* Add incompatibilities
 - As reported by the mod author on the mod's page:
 - Unique Landscapes - Cloudtop Mountains:
   https://www.nexusmods.com/oblivion/mods/16677/
 - Unique Landscapes - Colovian Highlands:
   https://www.nexusmods.com/oblivion/mods/15511/

* Add message
 - Requires disabled border regions.

sibir-ine commented on masterlist.yaml:
```diff
@@ -12708,6 +12708,28 @@ plugins:
+    req:
+      - *OBSE
+      - 'DLCShiveringIsles.esp'
+    inc:
```
The merged ESP includes these, so it can be added too:

```suggestion
    inc:
      - name: 'Unique Landscapes.esp'
        condition: 'version("Unique Landscapes.esp", "2.0.0", >=)'
```

sibir-ine committed:
Add incompatibility

- Unique Landscapes merged ESP:
 - https://www.nexusmods.com/oblivion/mods/19370/

sibir-ine reviewed (approved):
Thanks!
